roles/confluent.variables/vars/main.yml

```diff
@@ -1,6 +1,8 @@
 ---
 confluent_ansible_branch: 6.1.0-post
 
+systemd_base_dir: "{{'/lib/systemd/system' if ansible_distribution == 'Ubuntu' or ansible_distribution == 'Debian' else '/usr/lib/systemd/system'}}"
```

Comment:
i think theres an "ansible_family" var that might be equal to Debian that covers both ubuntu and debian

Reply:
Forgot about that var. Will change out.